### PR TITLE
ERSPAN support on Security Onion 14.04 through rcdcap

### PR DESCRIPTION
**Pull request by tlaurion**
# Description

ERSPAN decapsulation on SecurityOnion through rcdcap usage.
This include documentation to install rcdcap through Inverse repositoy and configuration so that rcdcap creates a new interface beofre nsm-sensor inspect traffic on newly create virtual interface.
# Delete branch after merge

YES 
# NEWS file entries

ERSPAN support on Security Onion: new package and repository with relative documentation.

### CODE REVIEW

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -2911,6 +2911,124 @@ Configuration of a new violation can use the following trigger types:
...
+
+  gpg --keyserver keyserver.ubuntu.com --recv 19CDA6A9810273C4
+  gpg --export --armor 19CDA6A9810273C4 | sudo apt-key add -
+
+ * Install rcdcap
+
+  sudo apt-get update
+  sudo apt-get install rcdcap  
+
+ * Create the file /etc/init.d/rcdcap with a similar content:
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: Wouldn't it be good to have the init script part of the package ?

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -2911,6 +2911,124 @@ Configuration of a new violation can use the following trigger types:
...
+ 
+  * Enable rcdcap service
+
+  sudo update-rc.d rcdcap defaults 18 02
+
+  * Start service
+
+  sudo /etc/init.d/rcdcap start
+
+  * Rerun Security Onion wizard to configure tweak mon1 interface and include it to be monitored. Attention: network configuration file will be overwritten. Backup if needed, or skip that part in the wizard and adapt manually.
```
Comment (julsemaan) [inline docs/PacketFence_Administration_Guide.asciidoc]: Not sure we can say `Attention: blablabla` in english. Most likely should be `Warning: Zammits cousin laundry service uses no name javel on your clothes`

And if you put `WARNING: Do not consume Zammits sandwiches if pregnant` on its on line its going to format it nicely when generating our doc

Just like magic but less spectacular.

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 3]
```diff
@@ -2911,6 +2911,68 @@ Configuration of a new violation can use the following trigger types:
   Type: suricata_event
   Trigger ID: The rule class of the triggered IDS alert
 
+ERSPAN
+++++++
+
+ERSPAN permits to mirror a local port traffic (low bandwidth) to a remote IP, E.G: your Security Onion already deployed box. ERSPAN encapsulates port traffic into ERSPAN then GRE and send that traffic to one/multiple destination(s). ERSPAN is a Cisco technology which is available only on some platforms, including: Catalyst 6500, 7600, Nexus, and ASR 1000.
+
+One way of accessing encapsulated traffic at the destination host is through a software called RCDCAP, which is a daemon that creates a virtual interface if not existing, on which both GRE and ERSPAN headers are decapsultated prior to the traffic being injected to the previous interface. Security Onion can then feed on that interface like it would on any other, and if the daemon dies, continueto listen to that interface even though decapsulated traffic won't be available anymore.
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: continueto ?! spanish ?

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 4]
```diff
@@ -2911,6 +2911,68 @@ Configuration of a new violation can use the following trigger types:
...
+ERSPAN
+++++++
+
+ERSPAN permits to mirror a local port traffic (low bandwidth) to a remote IP, E.G: your Security Onion already deployed box. ERSPAN encapsulates port traffic into ERSPAN then GRE and send that traffic to one/multiple destination(s). ERSPAN is a Cisco technology which is available only on some platforms, including: Catalyst 6500, 7600, Nexus, and ASR 1000.
+
+One way of accessing encapsulated traffic at the destination host is through a software called RCDCAP, which is a daemon that creates a virtual interface if not existing, on which both GRE and ERSPAN headers are decapsultated prior to the traffic being injected to the previous interface. Security Onion can then feed on that interface like it would on any other, and if the RCDCAP daemon dies, continue to listen to that interface even though decapsulated traffic won't be available anymore.
+
+
+Assomptions for the example:
+The switch is at IP 172.16.0.1, the monitored switch port is GigabitEthernet10 and the Security Onion monitoring destination IP is 10.10.10.10 on eth2, eth2 ideally being a dedicated interface.
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: it's more something like GigabitEthernet0/10 but it's a detail

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 5]
```diff
@@ -2911,6 +2911,68 @@ Configuration of a new violation can use the following trigger types:
...
+
+  * Rerun Security Onion wizard and make sure to skip network configuration step. Make sure that mon1 is selected for monitoring purposes, note that eth2 doesn't need to.
+   
+  sudo sosetup
+
+On the Switch:
+
+  monitor session 10 type erspan-source
+  description ERSPAN to 10.10.10.10
+  source interface GigabitEthernet10
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: Same here

**Pull request merged**